internal/envoy/bootstrap.go

```diff
@@ -56,6 +58,10 @@ type BootstrapConfig struct {
 	// Defaults to 8001.
 	XDSGRPCPort int
 
+	// XDSServerVersion defines the XDS Server Version to use.
+	// Defaults to "v2"
+	XDSServerVersion config.ServerVersion
```

Comment:
it'd be good to rename this one to XDSResourceVersion to match the flag.

Reply:
👍 

I'll rename `ServerVersion` in config.Parameters to this as well.